irods_browser_app/static/js/irods.js

```diff
@@ -0,0 +1,203 @@
+$('#btn-select-irods-file').on('click',function(){
+    $('#res_type').val($('#resource-type').val());
+    $('#file_struct').children().remove();
+	store = $('#root_store').val();
+	// Setting up the view tab
+	$('#file_struct').attr('name',store);
+	$('#irods_view_store').val(store);
+	// loading file structs
+	parent = $('#file_struct');
+	get_store(store,parent,0);
+	enable_settings();
+});
+
+function get_store(store,parent,margin){
+	margin_left = $(parent).css('margin-left');
+	margin_left = (parseInt(margin_left.substring(0,margin_left.length-2))-10) + 10;
+	form_data = new FormData();
+	form_data.append('store',store);
+	$.ajax({
+        mode: "queue",
+        url: '/irods/store/',
+        async: true,
+        type: "POST",
+        data: form_data,
+        processData: false,
+        contentType: false,
+        success: function (data, status) {
+            data = jQuery.parseJSON(data);
+            if (data[0].length ==0 && data[1].length ==0) {
+                $(parent).append("<div class='file' style='margin-left:"+margin+"px;'> -- EMPTY --</div>");
```

Comment:
this shouldn't be extracted from some other element. Just use the CSS to apply the same value to both elements.